ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -344,14 +345,14 @@ - (void) validatedAcquireTokenByRefreshToken:(NSString*)refreshToken
          resultItem.authority = _context.authority;
          
          
-         ADAuthenticationResult *result = [_context processTokenResponse:response forItem:resultItem fromRefresh:YES requestCorrelationId:_correlationId];
+         ADAuthenticationResult *result = [resultItem processTokenResponse:response fromRefresh:YES requestCorrelationId:_correlationId];
```

Comment:
Agree here :) It makes more sense to me to call processTokenResponse() through resultItem rather than _context.